components/web_server.rst

```diff
@@ -65,22 +71,54 @@ Configuration variables:
           auth:
             username: admin
             password: !secret web_server_password
+            
+    Example web_server configuration using version 2:
+
+    .. code-block:: yaml
 
-    Example web_server configuration with CSS and JS included from esphome-docs.
+        # Example configuration entry
+        web_server:
+          port: 80
+          version: 2
+          
+    Example web_server version 1 configuration with CSS and JS included from esphome-docs.
     CSS and JS URL's are set to empty value, so no internet access is needed for this device to show it's web interface.
     Force to turn off OTA function because the missing authentication.
 
     .. code-block:: yaml
 
-        # Example configuration entry
+        # Example configuration entry V1
         web_server:
           port: 80
           ota: false
           css_include: "../../../esphome-docs/_static/webserver-v1.min.css"
-          css_url: ""
           js_include: "../../../esphome-docs/_static/webserver-v1.min.js"
           js_url: ""
 
+    Example web_server version 2 configuration with JS included from a local file.
+    
+    CSS and JS URL's are set to empty value, so no internet access is needed for this device to show it's web interface.
```

Comment:
Can we put this in a section with "here by dragons" or something. We want most users just to enable the default web server and have it work.

Reply:
Yes - it's an advance option - you are better off using the defaults or even the `local` option that requires no external files

Reply:
re-arranged and simplified